kmir/Makefile

```diff
@@ -142,6 +142,7 @@ check-black: poetry-install
 
 kbuild-%: check-k-version
 	$(POETRY_RUN) kbuild kompile $*
+	@./build_glr.sh $* $(POETRY_RUN)
```

Comment:
This one might be a little confusing. Nix can't run poetry, so it when nix calls `make ...` it sets `POETRY_RUN=` so it is empty. This script will either be passed two args, the build target, and "poetry run" or nothing. And this will determine if it uses `poetry run` in the script.